core-patch/src/main/scala/eu/ostrzyciel/jelly/core/patch/handler/PatchHandler.scala

```diff
@@ -30,19 +30,21 @@ trait PatchHandler[-TNode]:
    * Add a namespace declaration to the patch stream.
    * This is called "prefix add" in RDF Patch. (PA)
    *
-   * @param name     the name of the namespace (without the trailing colon)
-   * @param iriValue the IRI value of the namespace
+   * @param name     the name of the namespace (without the trailing colon, required)
+   * @param iriValue the IRI value of the namespace (required)
+   * @param graph    the named graph to which the namespace belongs (optional)
    */
-  def addNamespace(name: String, iriValue: TNode): Unit
+  def addNamespace(name: String, iriValue: TNode, graph: TNode): Unit
```

Comment:
If the graph is optional, shouldn't there be an Option[TNode]?

Reply:
This would add overhead. We use nulls for that (TNode is even bounded by >: Null to ensure it's possible).